doc/caw.txt

```diff
@@ -368,6 +382,29 @@ A. Place "after/ftplugin/{filetype}/caw.vim" in your 'runtimepath'.
    We will support generating above ftplugin file from template file
    in the feature (TODO).
 
+					*caw-faq-3*
+Q. Why you renamed keymappings and variables?
+A. Because old keymappings and variables' names are hard to understand.
+   I changed old names like the followings:
+
+	Old		New ~
+	i		tildepos
+	I		zeropos
+	a		dollarpos
+
+	e.g.:
+	* |<Plug>(caw:i:toggle)| was changed to |<Plug>(caw:tildepos:toggle)|.
+	* |g:caw_I_sp_blank| was changed to |g:caw_zeropos_sp_blank|.
+
+	tildepos
+		The keymappings do actions for the position where ^ command moves to.
```

Comment:
@thinca makes me realize that this is hat, not tilde... :last_quarter_moon_with_face: